src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -605,27 +605,38 @@ private String getElementSrcWithRetryAfterWaitForPresence(By by) {
         }
     }
 
+    /**
+     * loads result of particular panel by giving panelId.
+     * @param panelId  the Id of panel element
```

Comment:
Should i write only what method is doing or you are referring to something else. Same style is followed in file line at line `300` and `342`.